lib/Core/Converter.cpp

```diff
@@ -245,8 +250,9 @@ std::list<ref<Rule> > Converter::getCondensedRules()
 std::string Converter::getVar(llvm::Value *V)
 {
     std::string name = V->getName();
+    name = "v" + name;
     std::ostringstream tmp;
-    tmp << "v_" << name;
```

Comment:
Please make this change dependent on t2output.  I am not completely sure, but some other parts of llvm2kittel may rely on the naming scheme.